frontend/alert/Dockerfile

```diff
@@ -9,15 +9,11 @@ COPY package.json /app
 COPY yarn.lock /app
 COPY alert/ /app/alert
 COPY shared-components/ /app/shared-components
-RUN yarn workspace alert install
-RUN yarn workspace pcx-shared-components install
+RUN yarn workspace penncoursealert install --production --frozen-lockfile
```

Comment:
This name has to match the name field in frontend/alert/package.json, so if we want to keep this for consistency's sake, we will have to update the name in package.json